### DISCUSSION TITLE
post/git-config-path-seperation

### DISCUSSION THREAD

**Discussion by giscus** [bbruceyuan/bbruceyuan.github.io#9] [Announcements]
# post/git-config-path-seperation

利用 Git 的 includeIf 配置，为不同的项目设置不同的配置，通过子文件夹 .gitconfig 覆盖 Git 的 Global 相关配置，比如 name，email 等。

https://bruceyuan.com/post/git-config-path-seperation.html

**Comment by OnlyPiglet** [comment]
写了个小工具，可以省点事情，偷点懒哈

https://github.com/OnlyPiglet/git-plus

https://freelifeblog.top/2024/07/01/git-plus工具推荐/


大佬可以看下 是不是契合😆
